docker/sirius/data/certificateProviderNoDecisionKBV.json

```diff
@@ -0,0 +1,177 @@
+{
+  "opg.poas.lpastore": {
+    "attorneys": [
+      {
+        "dateOfBirth": "1917-04-22",
+        "status": "removed",
+        "channel": "paper",
+        "uid": "3171d3d4-0404-b83e-6f63-c207a3edea6f",
+        "firstNames": "Keyon",
+        "lastName": "Jenkins",
+        "address": {
+          "line1": "875 Addison Turnpike",
+          "country": "FK",
+          "line3": "Tampa",
+          "postcode": "SZ5 2OR"
+        },
+        "email": "Americo0@gmail.com"
```

Comment:
Can you use example.com or similar emails, please.

Reply:
done